src/host/VtInputThread.cpp

```diff
@@ -94,7 +93,7 @@ VtInputThread::VtInputThread(_In_ wil::unique_hfile hPipe,
 DWORD WINAPI VtInputThread::StaticVtInputThreadProc(_In_ LPVOID lpParameter)
 {
     const auto pInstance = reinterpret_cast<VtInputThread*>(lpParameter);
-    return pInstance->_InputThread();
+    pInstance->_InputThread();

```

Comment:
FYI `_InputThread` is now `[[noreturn]]` because `CloseInput` is `[[noreturn]]`. Hence, no `return` is needed, despite this function "returning" a `DWORD`.